diff --git a/debian/libvirt-daemon-driver-secret.install.in b/debian/libvirt-daemon-driver-secret.install.in
index 41e010514..866741a46 100644
--- a/debian/libvirt-daemon-driver-secret.install.in
+++ b/debian/libvirt-daemon-driver-secret.install.in
@@ -1 +1,5 @@
 usr/lib/${DEB_HOST_MULTIARCH}/libvirt/connection-driver/libvirt_driver_secret.so
+etc/libvirt/secret.conf
+usr/lib/systemd/system/virt-secret-init-encryption.service
+usr/share/augeas/lenses/libvirt_secrets.aug
+usr/share/augeas/lenses/tests/test_libvirt_secrets.aug
diff --git a/debian/libvirt-l10n.install.in b/debian/libvirt-l10n.install.in
index 83f97c8d0..296e9ec2c 100644
--- a/debian/libvirt-l10n.install.in
+++ b/debian/libvirt-l10n.install.in
@@ -45,4 +45,5 @@ usr/share/locale/tr/LC_MESSAGES/libvirt.mo
 usr/share/locale/uk/LC_MESSAGES/libvirt.mo
 usr/share/locale/vi/LC_MESSAGES/libvirt.mo
 usr/share/locale/zh_CN/LC_MESSAGES/libvirt.mo
+usr/share/locale/zh_Hant/LC_MESSAGES/libvirt.mo
 usr/share/locale/zh_TW/LC_MESSAGES/libvirt.mo
diff --git a/debian/libvirt0.symbols b/debian/libvirt0.symbols
index 8689527f6..c14e61c2c 100644
--- a/debian/libvirt0.symbols
+++ b/debian/libvirt0.symbols
@@ -103,6 +103,7 @@ libvirt.so.0 libvirt0 #MINVER#
  *@LIBVIRT_10.2.0 10.2.0
  *@LIBVIRT_11.2.0 11.2.0
  *@LIBVIRT_PRIVATE_12.0.0 12.0.0
+ *@LIBVIRT_PRIVATE_12.1.0 12.1.0
 
 libvirt-qemu.so.0 libvirt0 #MINVER#
  *@LIBVIRT_QEMU_0.8.3 0.8.3
@@ -122,3 +123,4 @@ libvirt-admin.so.0 libvirt0 #MINVER#
  *@LIBVIRT_ADMIN_8.6.0 8.9.0
  *@LIBVIRT_ADMIN_11.2.0 11.2.0
  *@LIBVIRT_ADMIN_PRIVATE_12.0.0 12.0.0
+ *@LIBVIRT_ADMIN_PRIVATE_12.1.0 12.1.0
